src/app/components/Addcard/AddCard.module.css

```diff
@@ -0,0 +1,32 @@
+.card {
+  width: 100%;
+  display: grid;
+  grid-template-columns: 1fr 3fr 1fr;
+  color: var(--color-primary);
+}
+.imgage {
+  background-size: cover;
+  background-repeat: no-repeat;
+  border-radius: 0.5rem;
+  height: 3.75rem;
+  width: 3.75rem;
+}
+
+.content {
+  display: grid;
+  align-content: center;
+  padding-left: 1rem;
+}
+
+.addButton {
+  background-color: transparent;
+  background-repeat: no-repeat;
+  border: none;
+  cursor: pointer;
+  overflow: hidden;
+  outline: none;
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  stroke: var(--color-primary);
```

Comment:
For removing all the default button styles, you could use `all: unset` at the beginning and then just add the stuff you need under that.

Reply:
I have not thought about that. That slimifies my CSS a lot. Thank you!